internal/configs/virtualserver_test.go

```diff
@@ -1960,3 +1961,56 @@ func TestGenerateEndpointsForUpstream(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateSlowStartForPlusWithInCompatibleLBMethods(t *testing.T) {
+	serviceName := "test-slowstart-with-incompatible-LBMethods"
+
+	tests := []struct {
+		upstream conf_v1alpha1.Upstream
+		expected string
+	}{
+		{
+			upstream: conf_v1alpha1.Upstream{Service: serviceName, Port: 80, SlowStart: "10s"},
+			expected: "",
+		},
+	}
+
+	for _, test := range tests {
+		for lbMethod, _ := range incompatibleLBMethodsForSlowStart {
```

Comment:
this suggestion wasn't implemented https://github.com/nginxinc/kubernetes-ingress/pull/655#discussion_r316461167 please have a list of input parameters rather than taking lb methods from `incompatibleLBMethodsForSlowStart`